packaging/nmstate.spec

```diff
@@ -41,8 +41,9 @@ Requires:       python3-nispor
 Summary:        nmstate plugin for OVS database manipulation
 Requires:       python3-%{libname} = %{?epoch:%{epoch}:}%{version}-%{release}
 %if 0%{?rhel}
-# The python-openvswitch rpm package required only if openvswitch is installed.
+#The python-openvswitch rpm package is not in the same repo with nmstate, require only #if openvswitch is installed. If not installed, then it is recommended.
```

Comment:
Please, break the line as it was done before and keep the blank space between the comment and the `#`.